Documentation/gettingstarted/encryption-ipsec.rst

```diff
@@ -21,8 +21,14 @@ Packets are not encrypted when they are destined to the same node from which
 they were sent. This behavior is intended. Encryption would provide no benefits
 in that case, given that the raw traffic can be observed on the node anyway.
 
-Transparent encryption is not currently supported when chaining Cilium on top
-of other CNI plugins. For more information, see :gh-issue:`15596`.
+.. note::
```

Comment:
nit: Should we just put a `Limitations` section at the end, similar to other guides?

Also, do we need to call these out in `.. note::` sections? Developers commonly will add more and more of these boxes because they're shiny and blue, but it's often just a sign that the document isn't structured well enough to communicate the information clearly in dedicated sections. I think it probably makes sense to avoid flooding the user with a bunch of boxes at the start that describe all of the limitations, and instead first let the GSG describe the workflow and let the user try it out. Then, if they decide they will implement it, they can check the limitations section and understand any potential interactions with other features.

Reply:
Makes sense to me. I've pushed a new version.